include/ur_modern_driver/ros/controller.h

```diff
@@ -38,6 +38,7 @@ class ROSController : private hardware_interface::RobotHW, public URRTPacketCons
   ros::Time lastUpdate_;
   controller_manager::ControllerManager controller_;
   bool robot_state_received_;
+  ros::Time most_recent_packet_time_;
```

Comment:
I feel that you could reuse the `lastUpdate_` member variable

Reply:
since `lastUpdate_` is updated each time the `update` function is called and that function is also called, when no new package has been read, it does not quite fit the requirements.